### PR TITLE
[zh-cn] Sync content/zh-cn/docs/concepts/security/rbac-good-practices.md

### PR DESCRIPTION
**Pull request by jzhupup**
Sync upstream
```
content/zh-cn/docs/concepts/security/rbac-good-practices.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 1]
```diff
@@ -218,22 +222,14 @@ or other (third party) mechanisms to implement that enforcement.
...
+<!-- 
+For these reasons, namespaces should be used to separate resources requiring different levels of
+trust or tenancy. It is still considered best practice to follow [least privilege](#least-privilege)
+principles and assign the minimum set of permissions, but boundaries within a namespace should be
+considered weak.
 -->
-在命名空间中创建工作负载还会授予对该命名空间中 Secret 的间接访问权限。
-在 kube-system 或类似特权的命名空间中创建 Pod 
-可以授予用户不需要通过 RBAC 即可获取 Secret 访问权限。
+由于这些原因，命名空间应该被用来隔离不同级别的信任或租户的资源。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:231]:
```suggestion
出于这些原因，命名空间应该用于隔离不同的信任级别或不同租户所需的资源。
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 2]
```diff
@@ -218,22 +222,14 @@ or other (third party) mechanisms to implement that enforcement.
...
+遵循 [最小特权](#least-privilege) 原则并分配最小的权限集仍被认为是最佳实践，但命名空间的划分应被视为最小的权限集。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:232]:
```suggestion
遵循[最小特权](#least-privilege)原则并分配最小权限集仍被认为是最佳实践，但命名空间内的边界应视为弱。
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 3]
```diff
@@ -193,16 +193,20 @@ includes the contents of all Secrets.
...
+[ServiceAccount](/docs/reference/access-authn-authz/service-accounts-admin/), granting permission
+to create workloads also implicitly grants the API access levels of any service account in that
+namespace.
 -->
 ### 工作负载的创建 {#workload-creation}
 
-能够创建工作负载的用户（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)） 
-能够访问下层的节点，除非基于 Kubernetes 的
-[Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)做限制。
+在命名空间中创建工作负载的权限（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:206]:
```suggestion
在一个命名空间中创建工作负载（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)）的权限
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 4]
```diff
@@ -193,16 +193,20 @@ includes the contents of all Secrets.
...
+隐式地授予了对该命名空间中许多其他资源的访问权限，例如可以挂载在 Pod 中的 Secrets、ConfigMaps 和 PersistentVolume。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:207]:
```suggestion
隐式地授予了对该命名空间中许多其他资源的访问权限，例如可以挂载在 Pod 中的 Secret、ConfigMap 和 PersistentVolume。
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 5]
```diff
@@ -193,16 +193,20 @@ includes the contents of all Secrets.
...
+此外，由于 Pod 可以被任何 [服务账号](/zh-cn/docs/reference/access-authn-authz/service-accounts-admin/)
+运行，因此授予创建工作负载的权限也会隐式地授予该命名空间中任何服务帐号的 API 访问级别。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:209]:
```suggestion
此外，由于 Pod 可以被任何[服务账号](/zh-cn/docs/reference/access-authn-authz/service-accounts-admin/)运行，
因此授予创建工作负载的权限也会隐式地授予该命名空间中任何服务账号的 API 访问级别。
```

**Comment by tengqm** [comment]
Please read the [Chinese localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/) and revise accordingly.

/hold

**Comment by jzhupup** [comment]
> Please read the [Chinese localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/) and revise accordingly.
> 
> /hold

It has been revised, please check！

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 6]
```diff
@@ -82,9 +82,9 @@ some general rules that can be applied are :
   可以避免意外修改集群资源。
 - 避免将用户添加到 `system:masters` 组。任何属于此组成员的用户都会绕过所有 RBAC 权限检查，
   始终具有不受限制的超级用户访问权限，并且不能通过删除 `RoleBinding` 或 `ClusterRoleBinding`
-  来取消其权限。顺便说一句，如果集群是使用 Webhook 鉴权，此组的成员身份也会绕过该 
+  来取消其权限。顺便说一句，如果集群是使用 Webhook 鉴权，此组的成员身份也会绕过该
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:85]:
```suggestion
  来取消其权限。顺便说一句，如果集群使用 Webhook 鉴权，此组的成员身份也会绕过该
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 7]
```diff
@@ -187,22 +189,26 @@ includes the contents of all Secrets.
...
+[ServiceAccount](/docs/reference/access-authn-authz/service-accounts-admin/), granting permission
+to create workloads also implicitly grants the API access levels of any service account in that
+namespace.
 -->
 ### 工作负载的创建 {#workload-creation}
 
-能够创建工作负载的用户（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)） 
-能够访问下层的节点，除非基于 Kubernetes 的
-[Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)做限制。
+在一个命名空间中创建工作负载（Pod 或管理 Pod 的[工作负载资源](/zh-cn/docs/concepts/workloads/controllers/)）的权限隐式地授予了对该命名空间中许多其他资源的访问权限，例如可以挂载在
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:208]: 为什么不在中英文边界或标点符号处折行？

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md [hunk 8]
```diff
@@ -218,22 +224,14 @@ or other (third party) mechanisms to implement that enforcement.
...
+For these reasons, namespaces should be used to separate resources requiring different levels of
+trust or tenancy. It is still considered best practice to follow [least privilege](#least-privilege)
+principles and assign the minimum set of permissions, but boundaries within a namespace should be
+considered weak.
 -->
-在命名空间中创建工作负载还会授予对该命名空间中 Secret 的间接访问权限。
-在 kube-system 或类似特权的命名空间中创建 Pod 
-可以授予用户不需要通过 RBAC 即可获取 Secret 访问权限。
+出于这些原因，命名空间应该用于隔离不同的信任级别或不同租户所需的资源。
+遵循[最小特权](#least-privilege)原则并分配最小权限集仍被认为是最佳实践，但命名空间内的边界应视为弱。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/rbac-good-practices.md:234]:
```suggestion
遵循[最小特权](#least-privilege)原则并分配最小权限集仍被认为是最佳实践，
但命名空间内的边界概念应视为比较弱。
```

File: content/zh-cn/docs/concepts/security/rbac-good-practices.md (hunk 7 above)
Comment (jzhupup) [reply content/zh-cn/docs/concepts/security/rbac-good-practices.md]: 我换行这里涉及到的内容一直不是很熟练，后续我多学习，多看文档。

**Comment by windsonsea** [comment]
/unhold

due to revised and getting approved and lgtm.

**Pull request merged**